src/components/VideoDetailsSidebar/VideoDetailsSidebar.spec.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import VideoDetailsSidebar from './VideoDetailsSidebar.component';
+
+const mockData = {
```

Comment:
When you have mock data you can create another file and put it there, most of the time the mocked data won't change, on the other hand the tests are more likely to be updated and having a clean file make things easier to read and maintain